client/template.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
Consider moving the template to some other folder under client or even in tools as it is actually not part of client code.

Reply:
I was considering to take some time and explore making this tool a node module of sorts, so then these templates will live on within the 'CreateReactComponent' module files and allow projects to specify non-default templates with a `crc.config.json` file. Will adjust things around in another PR with issue #153